en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-latency.md

```diff
@@ -1,2 +1,27 @@
-!!! Note
-    Content to be added. WIP.
\ No newline at end of file
+# Latency Page
+![latency page full]({{base_path}}/assets/img/observe/latency/latency-page-full.png)
+
+The latency page latency stats of API calls that happened within the API management deployment. 
+Page will include summary view into the slowest APIs and then offer per API drill-down view to do further analysis.
+You can use this page as the starting point for debugging API slowness. See the topics given below to get details of the available widgets.
+
+## Page Walk Through
+The following topics walk you through the main features available for monitoring API latency.
+
+### Top 10 Slowest API Chart
+You can use line chart to figure out the slowest APIs of the API Management system at a glance. Since these are the APIs that contribute to the higher 95th percentile of the system, 
```

Comment:
```suggestion
Use line charts to find the slowest APIs of the API management system at a glance. Since these are the APIs that contribute to the higher 95th percentile of the system, improving these APIs lowers the 95th percentile of latency in the API Management system.
```